tests/api-controller-acceptance-tests/Jenkinsfile

```diff
@@ -54,11 +54,11 @@ podTemplate(label: label,
                         }
 
                         stage("install dependencies $application") {
-                            execute("make resolve", workDir, componentDir)
+                            execute("dep ensure --vendor-only -v", workDir, componentDir)
                         }
 
                         stage("code quality $application") {
```

Comment:
Please merge the "install dependencies" and "code quality" steps into one step that invokes the "before-commit.sh" script. We want to make sure no logic is executed directly by jenkinsfile.